skinlist.lua

```diff
@@ -2,7 +2,7 @@ local skins_dir_list = minetest.get_dir_list(skins.modpath.."/textures")
 
 for _, fn in pairs(skins_dir_list) do
 	local name, sort_id, is_preview, playername
-	local nameparts = string.gsub(fn, "[.]", "_"):split("_")
+	local nameparts = string.gsub(fn, "[.]", skins.fsep):split(skins.fsep)
```

Comment:
https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L3625

```Lua
local nameparts = string.gsub(fn, "[.]", "_"):split("_", 1)
```

should also fix the issue without a workaround like this, right?

Reply:
Could you explain the idea behind it? Because I gave it a try and didn't get any different result.

Reply:
That particular code splits `player_User_name` into `{ "player", "User_name" }` to properly separate the "player"/"character" part from the file name. I now see that the file extension would have to be split manually (either before this line or using index 2).

My idea is basically to provide a workaround for this issue without introducing a new format.

Reply:
> That particular code splits `player_User_name` into `{ "player", "User_name" }` to properly separate the "player"/"character" part from the file name.

Yep it splits into more than that.
```lua
filename = "playerORcharacter_accountname_skinnumberORname_optionalpreview.png"
local nameparts = string.gsub(filename, "[.]", "_"):split("_")
-- { [1] = playerORcharacter,[2] = accountname,[3] = skinnumberORname,[4] = optionalpreview,[5] = png,} 
```

>I now see that the file extension would have to be split manually (either before this line or using index 2).

File extension is already handled before splitting by replacing `.` with `_`.
```lua
string.gsub(fn, "[.]", "_")
```